noir-projects/noir-protocol-circuits/crates/rollup-lib/src/components.nr

```diff
@@ -146,6 +146,7 @@ pub fn compute_tx_effects_hash(combined: CombinedAccumulatedData) -> [Field; NUM
     // 1 unencrypted logs hash --> 1 sha256 hash -> 32 bytes  -> 2 fields | Beware when populating bytes that it is only 32!
     let mut txs_effects_hash_input = [0; TX_EFFECTS_HASH_INPUT_FIELDS];
 
+    let reverted = combined.reverted;
```

Comment:
This was the primary impetus for me simply adding the `reverted` code to `CombinedAccumulatedData`. This way we get the flag as an out from the private circuits as well.